# 2013-07-17 Stefan Teleman <stefan.teleman@oracle.com>
# This test case is marked as "expected to fail on Solaris", due to a
# misconfiguration of the GNU C++ Standard Library in Solaris.
--- gcc-4.7.3/libstdc++-v3/testsuite/26_numerics/headers/cmath/c99_classification_macros_c++0x.cc	2010-08-08 03:07:16.000000000 -0700
+++ gcc-4.7.3/libstdc++-v3/testsuite/26_numerics/headers/cmath/c99_classification_macros_c++0x.cc	2013-07-18 14:27:59.606294430 -0700
@@ -17,6 +17,8 @@
 
 // { dg-options "-std=gnu++0x" }
 // { dg-do compile { xfail uclibc } }
+// { dg-xfail-if "" { { *-*-solaris2.1[0-9]* } || { uclibc } } { "*" } { "" } }
+// { dg-excess-errors "" { target { { *-*-solaris* } || { uclibc } } } }
 // { dg-excess-errors "" { target uclibc } }
 
 #include <cmath>
